content/applications/sales/subscriptions/scheduled_actions.rst

```diff
@@ -0,0 +1,174 @@
+   Tools section --> Activate the developer mode`.
+
+With developer mode activated, navigate to :menuselection:`Settings app --> Technical --> Scheduled
+Actions`.
+
+.. image:: scheduled_actions/scheduled-actions-technical-settings-page.png
+   :align: center
+   :alt: The scheduled actions option under the technical menu in the Odoo Settings application.
+
+Doing so reveals a dedicated :guilabel:`Scheduled Actions` page. On this page, there is a complete
+list of scheduled actions for the entire database.
+
+From here, enter `Subscription` in the search bar. Doing so provides three subscription-specific
+results. The following documentation focuses on the last two results in the list:
+
+- :guilabel:`Sale Subscription: generate recurring invoices and payments`
+- :guilabel:`Sale Subscription: subscriptions expiration`
+
+.. image:: scheduled_actions/scheduled-actions-page-subscription-results.png
+   :align: center
+   :alt: The subscription-related results on the scheduled actions page in Odoo Settings.
+
+Ensure the scheduled actions are :guilabel:`Active`. If there is a ticked checkbox in the
+:guilabel:`Active` column for the desired subscription-related scheduled action on the
+:guilabel:`Scheduled Actions` page, that means they are `Active`.
+
+If a scheduled action needs to be activated, click into the desired scheduled action from the list.
+
+.. image:: scheduled_actions/scheduled-action-form.png
+   :alt: The scheduled action form in the Odoo Settings application.
+
+Then, from the scheduled action form, toggle the switch in the :guilabel:`Active` field to the
+right. Doing so turns the switch green, indicating that the scheduled action is now `Active`.
+
+The ability to set up how often the scheduled action runs is also available on the scheduled action
+form, in the :guilabel:`Execute Every` field.
+
+.. important::
+   The scheduled action will **not** function correctly if the execution time is less than five
+   minutes. This is a general rule for all scheduled actions.
+
+   For more information, read the :doc:`Frequent Technical Questions
+   </administration/odoo_sh/advanced/frequent_technical_questions>` documentation.
+
+Generate recurring invoices and payments
+========================================
+
+In order for the :guilabel:`Sale Subscription: generate recurring invoices and payments` scheduled
+action to run properly, *Deferred Expense* and *Deferred Revenue* accounts **must** set up.
+
+To set up *Deferred Expense* and *Deferred Revenue* accounts, navigate to :menuselection:`Accounting
+app --> Configuration --> Settings`. Both accounts can be configured in the :guilabel:`Default
+Accounts` section.
+
+.. image:: scheduled_actions/deferred-settings-accounting.png
+   :align: center
+   :alt: The necessary deferred account settings in the Odoo Accounting app's settings page.
+
+Once the necessary information and configurations are entered in the :guilabel:`Deferred Expense`
```

Comment:
what "necessary information and configurations," specifically?

Reply:
I removed that phrase and replaced it with a more specific mention of the correct accounts needing to be entered in those drop-down menu fields.